src/content/core/user-roles-and-permissions.mdx

```diff
@@ -0,0 +1,129 @@
+This page outlines the roles, permissions, and key features available to Admins and Developers in Okteto.
+
+## User Roles
+
+In Okteto, there are two primary roles: **Admins** and **Developers**.
+
+## Admins
+
+Admins have extensive control and oversight capabilities across the Okteto platform. Key features include:
+
+### Admin Access to Any Environment
+Admins can perform all operations on any Namespace or Preview Environment, including:
+- Deleting, sharing, waking, sleeping, and marking Namespaces or Preview Environments as persistent
+- Deploying or deleting applications using both Okteto and `kubectl`
+- Accessing all resources and sub-resources within any Development Environment
+- [Transferring Namespace ownership](core/namespaces.mdx#transfer-namespace-ownership) between developers
+- Fine tuning Developer access to Namespaces and Preview Environments by configuring [the `serviceAccounts` section within the Okteto Helm configuration](self-hosted/helm-configuration.mdx#serviceaccounts).
+
+
+**Benefits**: This level of access allows admins to manage and troubleshoot environments, even if the original owner is unavailable.
+
+## Access to the Okteto Admin Dashboard
+Admins can manage their instance through the [Okteto Admin Dashboard](admin/dashboard.mdx), including:
+- [Managing all users](admin/dashboard.mdx#users) within their Okteto instance
+- Having an overview of [Kubernetes Node utilization](admin/dashboard.mdx#nodes)
+- Seeing and managing all [Namespaces and Preview Environments](admin/dashboard.mdx#namespaces) within their Okteto instance
+- Setting cluster-wide variables in the [Admin Variables section](admin/dashboard.mdx#admin-variables)
+- Configuring [Admin Access Tokens](admin/dashboard.mdx#admin-access-tokens) for shared automations
+- Creating a library of ready-to-use Environments in the [Catalog](admin/catalog.mdx)
+
+
+## Developers
+
+Developers have permissions primarily focused on creating and managing their own environments. Key features include:
+
+### Creating and Managing Environments
+Developers can:
+- Create Namespaces and Personal/Global Preview Environments
+- Delete Namespaces and Preview Environments they own
+- Share Namespaces and Preview Environments they own
+- Deploy applications using both Okteto and `kubectl` within their own or shared Namespaces and Previews Environments
+- Wake and sleep Namespaces and Preview Environments they own or have access to
+
+### Access to Global Preview Environments
+Developers have read access to Global Preview Environments, enabling them to troubleshoot issues effectively using `kubectl`.
+
+**Benefits**: This feature allows developers to monitor and debug shared environments without requiring elevated permissions.
+
+## Permission Model
+
+### Namespace Operations
+
+| Action | Admin | Developer-Owner | Developer-Member | Developer without direct access |
```

Comment:
I just realize, should we add a legend or similar explaining what means each symbol? I think at least we should mention the `-`, it might not be clear for readers

Reply:
Good call! Updated this

Reply:
It looks really good! 👏